cmd/mailbox-controller/controller.go

```diff
@@ -144,13 +146,18 @@ func (ctl *mbCtl) enqueue(obj any) {
 		logger.V(4).Info("Enqueuing reference due to workspace", "wsName", typed.Name)
 		ctl.queue.Add(typed.Name)
 	case *edgev2alpha1.SyncTarget:
-		mbwsName := ctl.mbwsNameOfSynctarget(typed)
-		if mbwsName == "" {
-			logger.Error(nil, "Failed to construct mailbox workspace name from SyncTarget", "syncTargetName", typed.Name)
-			return
+		mbwsName, err := ctl.mbwsNameOfSynctarget(typed)
+		if err != nil {
```

Comment:
maybe changing the condition order will be more clear ? 
If (err) {
  If != errNoSpaceId {
    return
    }
    ctl.queue.Add(obj)
 } 
 else {
 }

Reply:
I didn't want to return from switch, maybe someone will add some code after the switch-case.